README.md

```diff
@@ -1 +1,2 @@
-# inventor-be
\ No newline at end of file
+# inventor-be
+https://github.com/cypherkrescent/inventor-be.wiki.git
```

Comment:
Your link is invalid. 

- Use hyperlink.
- The link should point to your architectural design pattern on wiki page.